tests/libyui/menu_item_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "rspec_tmux_tui"
+
+describe "Menu Item" do
+  around(:each) do |ex|
```

Comment:
In this case it's only two tests, so it shouldn't matter that much. But in general: Is it a good idea to restart the program for every test? As tests accumulate, this will be a heavy performance hit.

In many cases, this one included, we can easily start the program once, execute many tests, and terminate it after the last one; provided, of course, that the individual tests are independent enough not to change the internal state of the test program too much.

Let's discuss this later today or maybe tomorrow.

Reply:
OK, good point. When we add new tests we should do it without program restarting, just to make sure the library makes it easy.